src/repository/participant-repository.ts

```diff
@@ -0,0 +1,74 @@
+import { Participant } from "../model/participant";
+import { ParticipantParams } from "../util/types";
+import { Company } from "../model/company";
+
+export class ParticipantRepository {
+  async createParticipant(
+    participantParams: ParticipantParams,
+    companyId: number
+  ) {
+    try {
+      const company = await Company.findOne({
+        where: { company_id: companyId },
+      });
+      if (company) {
+        return await Participant.create({
+          name: participantParams.name, 
```

Comment:
shorthand syntax you can use spread operator ex. ->  return await Participant.create({
  ...participantParams,
  company,
}).save();

Reply:
Changed it on every create method.